lib/perl/Genome/Model/Tools/FastTier/TierBitmasks.pm

```diff
@@ -116,174 +61,67 @@ sub create {
-    #in_place_difference_genomes($regulatory_regions, $tier2);
-    print STDERR "Calculated Tier2 regulatory regions / repeatmasker\n";
-    $self->debug_message("Calculated Tier2 regulatory regions / repeatmasker\n");
-    #printf "Tier2 regulatory set encompasses %u bases. %f%% of the genome\n", bases_covered($regulatory_regions), bases_covered($regulatory_regions)/$masked_genome_size * 100;
-
-    $self->in_place_union_genomes($tier2, $regulatory_regions);
-    print STDERR "Calculated Tier2 conserved U regulatory regions / repeatmasker\n";
-    $self->debug_message("Calculated Tier2 conserved U regulatory regions / repeatmasker\n");
-    $self->in_place_difference_genomes($tier2, $tier1); #exclude things hitting Tier1
-    print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n";
-    $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1\n");
-    $self->in_place_difference_genomes($tier2, \%genome); #account for masking
-    print STDERR "Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n";
-    $self->debug_message("Calculated (Tier2 conserved U regulatory regions / repeatmasker) / Tier1 / masked genome\n");
-    printf "Tier2 encompasses %u bases. %f%% of the genome\n", $self->bases_covered($tier2), $self->bases_covered($tier2)/$masked_genome_size * 100;
+    printf $self->report_coverage("Tier2", $tier2, $masked_genome_size);
     $self->write_genome_bitmask($self->temp_staging_directory."/tier2.bitmask", $tier2);
-    #free up some mem?
-    undef($regulatory_regions);
 
     #Tier3
     #Tier3 contains all remaining non-repeat bases
     $self->in_place_union_genomes($tier2, $tier1); #everything in tier1 and tier2
     my $tier3 = $self->complement_genome($tier2); #Everything not in tier1 or tier2
-    $self->in_place_difference_genomes($tier3,$repeatmasker_regions); #should be everything not in tier1 or tier2 and not in regulatory regions
+    $self->in_place_difference_genomes($tier3, $repeatmasker_regions); #should be everything not in tier1 or tier2 and not in regulatory regions
     $self->in_place_difference_genomes($tier3, \%genome);
     $self->write_genome_bitmask($self->temp_staging_directory."/tier3.bitmask", $tier3);
-    #free up some mem?
-    printf "Tier3 encompasses %u bases. %f%% of the genome\n", $self->bases_covered($tier3), $self->bases_covered($tier3)/$masked_genome_size * 100;
+
+    printf $self->report_coverage("Tier3", $tier3, $masked_genome_size);
     my $tier4 = $self->union_genomes($tier2, $tier3);
 
     ($tier1,$tier2,$tier3) = (undef,undef,undef);
 
     $self->in_place_complement_genome($tier4);
     $self->in_place_difference_genomes($tier4, \%genome);
-    printf "Tier4 encompasses %u bases. %f%% of the genome\n", $self->bases_covered($tier4), $self->bases_covered($tier4)/$masked_genome_size * 100;
+    printf $self->report_coverage("Tier4", $tier4, $masked_genome_size);
```

Comment:
This should now be a `print` instead of `printf` since the formatting is handled inside `report_coverage`.